### DISCUSSION TITLE
how to use filter to get messages from a user with a blank in the name?

### DISCUSSION THREAD

**Discussion by FortunateBliss** [Tyrrrz/DiscordChatExporter#995] [General]
Hello,
I am having trouble using the filter in the GUI for usernames with a blank e.g.:
from:John Doe#1234
I have tried those as well:
"from:John Doe#1234"
from:JohnDoe#1234
from:John_Doe#1234

does this work at all? if so in what format?

the entire search term usually cannot have any blanks. by default in discord itself there is always a blank between from: name  and other filters. though those have to be removed.

maybe it can be programmed more compatible so that i can just copy and paste the exact search term from the discord windows in there including all blanks

**Comment by 96-LB** [comment]
Use quotes: `from:"John Doe#1234"` or `from:'John Doe#1234'`.

Alternatively, use their user ID: `from:1234567890987654321`.
